SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialVirtualWorkerTranslator.cpp

```diff
@@ -12,36 +10,14 @@ DEFINE_LOG_CATEGORY(LogSpatialVirtualWorkerTranslator);
 
 SpatialVirtualWorkerTranslator::SpatialVirtualWorkerTranslator(UAbstractLBStrategy* InLoadBalanceStrategy,
 	USpatialStaticComponentView* InStaticComponentView,
```

Comment:
probably being dumb and missing it but I can't see any usage of the static component view anymore?

Reply:
Great catch! I'll happily remove it.